src/lib/sql/sql_translator.cpp

```diff
@@ -654,6 +648,32 @@ std::shared_ptr<AbstractLQPNode> SQLTranslator::_translate_stored_table(
   return validated_stored_table_node;
 }
 
+std::shared_ptr<AbstractLQPNode> SQLTranslator::_translate_meta_table(
+    const std::string& name, const std::shared_ptr<SQLIdentifierResolver>& sql_identifier_resolver) {
+  AssertInput(MetaTableManager::is_meta_table_name(name), std::string{"Did not find a meta table with name "} + name);
+
+  // MetaTables are non-cacheable because they might contain information about the general system state
+  // that can change at any time
+  _cacheable = false;
+
+  const auto meta_table_name = name.substr(MetaTableManager::META_PREFIX.size());
+  const auto meta_table = Hyrise::get().meta_table_manager.generate_table(meta_table_name);
```

Comment:
Please describe why we are doing this here even though it does not really belong to the translation.